recipes-navq/u-boot-env-setup/u-boot-env-setup.bb

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2023 VoxelBotics
+# U-boot env helper service
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = " \
+	file://u-boot-env-setup.sh \
+	file://u-boot-env-setup.service \
+"
+
+inherit systemd
+
+do_install() {
+	install -d ${D}${sysconfdir}
+	install -d ${D}${sbindir}
+	install -d ${D}${systemd_system_unitdir}
+
+	echo "/dev/mmcblk1	0x400000	0x4000" > ${D}${sysconfdir}/fw_env.config_sd
+	echo "/dev/mmcblk2	0x400000	0x4000" > ${D}${sysconfdir}/fw_env.config_emmc
+
+	install -m 0755 ${WORKDIR}/u-boot-env-setup.sh ${D}${sbindir}
+	install -m 0644 ${WORKDIR}/u-boot-env-setup.service ${D}${systemd_system_unitdir}
+}
+
+SYSTEMD_AUTO_ENABLE = "enable"
+SYSTEMD_SERVICE:${PN} = "u-boot-env-setup.service"
+FILES:${PN} += "${systemd_system_unitdir}"
+INSANE_SKIP:${PN} += "file-rdeps"
```

Comment:
Is this really needed?

Reply:
You mean insane skip? bitbake detects that the shell script is using bash, and wants that dependency to be set in the bb script. But the bash package comes from the Ubuntu layer, and cannot be listed as RDEPENDS. So yes, we need insane_skip to get rid of the error with bash.

Reply:
Ok, after last conversation I've removed dependency on bash. So no need for insane_skip anymore. Please take a loook.